packages/terra-graphs/src/terra-dev-site/doc/Graphs/Graphs.1.doc.mdx

```diff
@@ -0,0 +1,36 @@
+import { Badge } from '@cerner/terra-graphs/package.json?dev-site-package';
+
+import DefaultGraphs from '../example/DefaultGraphs?dev-site-example';
+
+import PropsTable from '@cerner/terra-graphs/src/Graphs?dev-site-props-table';
+
+<Badge />
+
+# Terra Graphs
+
+{insert description}
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-graphs`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Graphs from 'terra-graphs';
+
+<Graphs {props...} />
+```
```

Comment:
We can slim this down to only the import. The src code can be view in the example(s)